src/Razor/Directory.Build.props

```diff
@@ -3,6 +3,11 @@
 
   <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.props', '$(MSBuildThisFileDirectory)../'))" />
 
+  <PropertyGroup>
+    <AnalysisLevel>Latest</AnalysisLevel>
```

Comment:
I was having a lot of trouble getting the Analyzer to run on my machine, these properties seemed to help but I have my doubts since I think they should have already been in effect anyway. If someone on a (hopefully not haunted) different machine can check if these are necessary that would be great.

Reply:
@jmarolf is usually the wizard in this area

Reply:
They didn't end up being required, but I'm still not sure why my machine was acting up Friday.